torchbenchmark/util/model.py

```diff
@@ -0,0 +1,22 @@
+import json
+import os
+import pandas as pd
+import typing
+from  collections.abc import Iterable
+import torch
+
+class BenchmarkModel():
+    def __init__(self, *args, **kwargs): 
```

Comment:
It'd be better to finish this implementation - e.g.
- fill in the train and eval methods with raise NotImplemented
- set a docstring at least pointing to the place where we define this interface in .md file (maybe later we move the comments to inline and generate doc but i'd rather not duplicate the same doc in 2 places for now)